### ISSUE TITLE
Ensure that semanticdb is preserved for all projects in sbt-scalahost

### ISSUE THREAD

**Issue by olafurpg** [scalameta/scalameta#685]
Currently, compiling two projects in parallel can cause one project to override the semanticdb of another project.

**Comment by xeno-by** [comment]
Duplicate of https://github.com/scalameta/scalameta/issues/650

**Comment by olafurpg** [comment]
Aha, I didn't make the connection that these were related, thanks!
